core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/action/ActionConfiguration.java

```diff
@@ -152,7 +153,7 @@ public void insert(Action action) {
                                 .orElseThrow(RuntimeException::new)) + "," +
                 SQLTools.getStringForSQL(action.getRetries()) + "," +
                 SQLTools.getStringForSQL(action.getErrorExpected()) + "," +
-                SQLTools.getStringForSQL(action.getErrorStop()) + ");";
+                SQLTools.getStringForSQL(action.getErrorStop()) + ", 'NA' );";
```

Comment:
doesn't ActionKey contain deletedAt? don't we need to use this value?

Reply:
Yes ActionKey contains deletedAt but for non-deleted record we should explicitly write 'NA'.